fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/logical/LogicalFileScan.java

```diff
@@ -149,6 +149,13 @@ public SelectedPartitions(long totalPartitionNum, Map<Long, PartitionItem> selec
             }
         }
 
+        /**
+         * Return true means this relation is partition prunned
+         */
+        public boolean isPartitionPruned() {
+            return this.totalPartitionNum > 0 || isPartitionPruned;
```

Comment:
need to explain in comment why `this.totalPartitionNum > 0` means isPartitionPruned

Reply:
Changed, the previous code is confusing